### ISSUE TITLE
Docker CLI button not working

### ISSUE THREAD

**Issue by cosmin-harangus** [kitematic/kitematic#563]
Just updated this morning to the latest version of Kitematic and now the Docker CLI button does nothing.

**Comment by DieterReuter** [comment]
I just updated to Kitematic 0.6.2 and it's working fine for me on OSX. The Docker CLI button opens up a terminal window (I'm using iTerm) and I can use `docker ps` commands and all as expected.

**Comment by paulm-jones** [comment]
I am having the same problem with with 0.6.0 and 0.6.2.

If I click the new 'Docker CLI' button or File > Open...Terminal

    Unhandled rejection Error: /Applications/Kitematic 0.6.2.app/Contents/Resources/app/resources/terminal
	DOCKER_HOST=tcp://192.168.99.100:2376
	DOCKER_CERT_PATH=/Users/paulj/.docker/machine/machines    /dev
	DOCKER_TLS_VERIFY=1 $SHELL returned non zero exit code.
	Stderr: 464:464: syntax error: Expected     “"” but found end of script. (-2741)

        at /Applications/Kitematic 0.6.2.app/Contents/Resources/app/build/utils/Util.js:16:18
        at ChildProcess.<anonymous> (/Applications/Kitematic 0.6.2.app/Contents/Resources/app/    node_modules/exec/exec.js:91:5)
        at emitTwo (events.js:87:13)
        at ChildProcess.emit (events.js:169:7)
        at maybeClose (child_process.js:996:16)
        at Socket.<anonymous> (child_process.js:1165:11)
        at emitOne (events.js:77:13)
        at Socket.emit (events.js:166:7)
        at Pipe._onclose (net.js:464:12)

This is on Yosemite 10.10.3. I had a dig into the app bundle to see if I could spot the problem but couldn't find anything obvious (to me) - is there anything I can provide to help?

**Comment by goloroden** [comment]
Same problem here ... 0.6.2 on OS X Yosemite.

**Comment by robinduckett** [comment]
+1

**Comment by northerngit** [comment]
I've submitted a pull request to fix this issue - https://github.com/kitematic/kitematic/issues/565

In the mean time, you can manually implement the fix.

First, close Kitematic and from the OS X Terminal:

```
sudo curl -o /Applications/Kitematic\ \(Beta\).app/Contents/Resources/app/resources/terminal https://raw.githubusercontent.com/northerngit/kitematic/master/resources/terminal
```

Finally, ensure the file is executable.

```
sudo chmod +x /Applications/Kitematic\ \(Beta\).app/Contents/Resources/app/resources/terminal
```

Run Kitematic and everything should be back to normal!

**Comment by robinduckett** [comment]
Nice one! Thanks a lot

**Comment by jeffdm** [comment]
Thanks @northerngit ! Fixed in 0.6.3.
